doc/specs/stdlib_math.md

```diff
@@ -275,3 +275,75 @@ program demo_logspace_rstart_cbase
 
 end program demo_logspace_rstart_cbase
 ```
+## `arange`
+
+### Status
+
+Experimental
+
+### Class
+
+Pure function.
+
+### Description
+
+Create a vector of the `integer/real` type with given fixed spaced values within a given interval.
+
+#### Note
+
+Because of the `i` (`huge(integer :: i) = 2147483647`) index inside the `arange` function , the dimensional maximum length of array created by the `arange` function is `2147483647`. 
```

Comment:
Is there a use case to create larger arrays? I can't think of one, but if there is, we could use `int64` to allow larger arrays.

Reply:
I think the length of `2147483647` as an array dimension is very sufficient. `int64` seems unnecessary, but I may subconsciously explain this in the document. Does it need to be removed?

Reply:
Okay, great, I was just wondering. I think it's okay that it stays int32. Also, the note is helpful IMO.